roles/frr/templates/frr_leaf.conf.j2

```diff
@@ -65,3 +65,23 @@ exit
 ip protocol bgp route-map RM_SET_SRC4
 !
 ipv6 protocol bgp route-map RM_SET_SRC6
+!
+{% for vrrp_group in frr_vrrp_groups %}
+interface {{ vrrp_group.interface }}
+ vrrp {{ vrrp_group.vrid }}{% if vrrp_group.ipv6 | default([]) | length > 0 %} version 3{% endif %}
```

Comment:
`version 3` is the default and can be used for both v4+v6, no need to add the extra logic